src/plugins/sqlite_persistence_plugin/sqlite_persistence.cpp

```diff
@@ -1,9 +1,13 @@
 #define BRIDGE_PLUGIN
 #include "persistence_plugin.h"
 #include "sqlite3.h"
-#include <mutex>
 #include <nlohmann/json.hpp>
```

Comment:
Any use separating out these headers like we do elsewhere?

Reply:
I might space them out when I to my touch-up pass on this repo. For now though I think this is fine.